commit d3b5c4cd4d90174355820fa77dabe00959992c01
Author: Gu Zhongyan <guzhongyan@360.cn>
Date:   Thu Mar 29 15:44:25 2018 +0800

    misc: fix various log messages
    
    add space between __func__ and following messages
    
    Signed-off-by: Gu Zhongyan <guzhongyan@360.cn>

diff --git a/src/mgr/StandbyPyModules.cc b/src/mgr/StandbyPyModules.cc
index 5430658bf10..756b8be7429 100644
--- a/src/mgr/StandbyPyModules.cc
+++ b/src/mgr/StandbyPyModules.cc
@@ -170,7 +170,7 @@ bool StandbyPyModule::get_config(const std::string &key,
   const std::string global_key = PyModuleRegistry::config_prefix
     + get_name() + "/" + key;
 
-  dout(4) << __func__ << "key: " << global_key << dendl;
+  dout(4) << __func__ << " key: " << global_key << dendl;
 
   return state.with_config([global_key, value](const PyModuleConfig &config){
     if (config.count(global_key)) {
